File: my-folder/0088-merge-sorted-array/solution.js
/**
 * @param {number[]} nums1
 * @param {number} m
 * @param {number[]} nums2
 * @param {number} n
 * @return {void} Do not return anything, modify nums1 in-place instead.
 */
var merge = function(nums1, m, nums2, n) {
  let i = m


    for(let j=0; j<n; j++){
        nums1[i] = nums2[j]
        i++
    }

  nums1.sort((a,b) => a-b)
    
};

File: my-folder/0088-merge-sorted-array/solution.rs
impl Solution {
    pub fn merge(nums1: &mut Vec<i32>, m: i32, nums2: &mut Vec<i32>, n: i32) {
        let mut i = m as usize;

        for &num in nums2.iter() {
            nums1[i] = num;
            i+=1;
        }

        nums1.sort()
    }
}
